src/app/views/activities-and-events/activities-and-events.component.html

```diff
@@ -1,101 +1,40 @@
+          </div>
+        </div>
+      </div>
+    </div>
   </div>
-  <img class="col-md-5 col-12 offset-md-1 py-2"  src="../../../assets/images/Monsoon School/Monsoon-School_2017_Group_photo.jpg">
-
-
 </div>
 
-      <!-- -----------Section - 4 --------------------- -->
-
-<!-- <div class=" justify-content-center align-items-center p-md-5 m-md-5 p-2 " style="background-color: rgba(212, 212, 212, 0.369);">
-  <h3 class="bold" style="font-size: 50px;">Student Blogs / Media</h3>
-  <div class="card align-self-center p-5 m-1 shadow my-2 ">
-    s in a piece of classical Latin literature from 45 BC, making it over 2000 years old. Richard McClintock, a Latin professor at Hampden-Sydney College in Virginia, looked up one of the more obscure Latin words, consectetur, from a Lorem Ipsum passage, and going through the cites of the word in classical literature, discovered the undoubtable source. Lorem Ipsum comes from sections 1.10.32 and 1.10.33 of "de Finibus Bonorum et Malorum" (The Extremes of Good and Evil) by Cicero, written in 45 BC. This book is a treatise on the theory of ethics, very popular during the Renaissance. The first line of Lorem Ipsum, "Lorem ipsum dolor sit amet..", comes from a line in section 1.1<br>~ Archit
-    <a href="">Read more..</a>
-  </div>
-  <div class="card align-self-center p-5 m-1 shadow my-2 ">
-    s in a piece of classical Latin literature from 45 BC, making it over 2000 years old. Richard McClintock, a Latin professor at Hampden-Sydney College in Virginia, looked up one of the more obscure Latin words, consectetur, from a Lorem Ipsum passage, and going through the cites of the word in classical literature, discovered the undoubtable source. Lorem Ipsum comes from sections 1.10.32 and 1.10.33 of "de Finibus Bonorum et Malorum" (The Extremes of Good and Evil) by Cicero, written in 45 BC. This book is a treatise on the theory of ethics, very popular during the Renaissance. The first line of Lorem Ipsum, "Lorem ipsum dolor sit amet..", comes from a line in section 1.1<br>~ Archit
-    <a href="">Read more..</a>
-  </div>
-  <div class="card align-self-center p-5 m-1 shadow my-2 ">
-    s in a piece of classical Latin literature from 45 BC, making it over 2000 years old. Richard McClintock, a Latin professor at Hampden-Sydney College in Virginia, looked up one of the more obscure Latin words, consectetur, from a Lorem Ipsum passage, and going through the cites of the word in classical literature, discovered the undoubtable source. Lorem Ipsum comes from sections 1.10.32 and 1.10.33 of "de Finibus Bonorum et Malorum" (The Extremes of Good and Evil) by Cicero, written in 45 BC. This book is a treatise on the theory of ethics, very popular during the Renaissance. The first line of Lorem Ipsum, "Lorem ipsum dolor sit amet..", comes from a line in section 1.1<br>~ Archit
-    <a href="">Read more..</a>
-  </div>
-</div> -->
-
+<button class="scroll-to-top" #scrollToTopButton (click)="scrollToTop()">↑</button>
```

Comment:
We do not need this #scrollToTopButton. 
Just add in class "d-none d-md-block"

Reply:
Done. Everywhere.